### DISCUSSION TITLE
About tracing issues in macos system【Exporter otlp encountered the following error(s): the grpc server returns error (Unknown error): , detailed error message: transport error】

### DISCUSSION THREAD

**Discussion by 0x0918** [open-telemetry/opentelemetry-rust#1923] [General]
I tried running tracing-jaeger in examples。
<img width="1356" alt="image" src="https://github.com/open-telemetry/opentelemetry-rust/assets/90438284/e4d7d613-9b73-498a-b424-57e7b0f8a8eb">
docker run is：```run -d -p 16686:16686 -p 4317:4317 -e COLLECTOR_OTLP_ENABLED=true jaegertracing/all-in-one:latest ```
<img width="767" alt="image" src="https://github.com/open-telemetry/opentelemetry-rust/assets/90438284/fc1a01c6-d56a-4460-a657-c2b2a5da13c8">
pls help me

**Comment by utpilla** [comment]
@0x0918 Could you please try this with the latest version of `opentelemetry-otlp` crate? We recently made some changes to have more informative errors logged for these scenarios. Here's the PR for reference: #1931
